zerver/models.py

```diff
@@ -874,61 +874,61 @@ def __str__(self) -> str:
         return f"<RealmFilter({self.realm.string_id}): {self.pattern} {self.url_format_string}>"
 
 
-def get_realm_filters_cache_key(realm_id: int) -> str:
-    return f"{cache.KEY_PREFIX}:all_realm_filters:{realm_id}"
+def get_linkifiers_cache_key(realm_id: int) -> str:
+    return f"{cache.KEY_PREFIX}:all_linkifiers_for_realm:{realm_id}"
 
 
 # We have a per-process cache to avoid doing 1000 remote cache queries during page load
-per_request_realm_filters_cache: Dict[int, List[Tuple[str, str, int]]] = {}
+per_request_linkifiers_cache: Dict[int, List[Tuple[str, str, int]]] = {}
 
 
-def realm_in_local_realm_filters_cache(realm_id: int) -> bool:
-    return realm_id in per_request_realm_filters_cache
+def realm_in_local_linkifiers_cache(realm_id: int) -> bool:
+    return realm_id in per_request_linkifiers_cache
 
 
-def realm_filters_for_realm(realm_id: int) -> List[Tuple[str, str, int]]:
-    if not realm_in_local_realm_filters_cache(realm_id):
-        per_request_realm_filters_cache[realm_id] = realm_filters_for_realm_remote_cache(realm_id)
-    return per_request_realm_filters_cache[realm_id]
+def linkifiers_for_realm(realm_id: int) -> List[Tuple[str, str, int]]:
+    if not realm_in_local_linkifiers_cache(realm_id):
+        per_request_linkifiers_cache[realm_id] = linkifiers_for_realm_remote_cache(realm_id)
+    return per_request_linkifiers_cache[realm_id]
 
 
-@cache_with_key(get_realm_filters_cache_key, timeout=3600 * 24 * 7)
-def realm_filters_for_realm_remote_cache(realm_id: int) -> List[Tuple[str, str, int]]:
+@cache_with_key(get_linkifiers_cache_key, timeout=3600 * 24 * 7)
+def linkifiers_for_realm_remote_cache(realm_id: int) -> List[Tuple[str, str, int]]:
     filters = []
-    for realm_filter in RealmFilter.objects.filter(realm_id=realm_id):
-        filters.append((realm_filter.pattern, realm_filter.url_format_string, realm_filter.id))
+    for linkifier in RealmFilter.objects.filter(realm_id=realm_id):
+        filters.append((linkifier.pattern, linkifier.url_format_string, linkifier.id))
 
     return filters
 
 
-def all_realm_filters() -> Dict[int, List[Tuple[str, str, int]]]:
+def linkifiers_from_all_realms() -> Dict[int, List[Tuple[str, str, int]]]:
```

Comment:
I think I like `all_linkifiers_for_installation`  better as a name. -- we sometimes use "installation" as the term for an entire Zulip server in the backend.

Reply:
Done, renamed to `all_linkifiers_for_installation`.